.travis.yml

```diff
@@ -63,25 +58,15 @@ before_install:
 
     git config --global advice.detachedHead false
 
-    # Export tests environment variables
-    source tests/_ci/environment
-    export $(cut -d= -f1 tests/_ci/environment)
-
-    # Export build environment variables
-    export PHP_MAJOR="$(php -r 'echo phpversion();' | cut -d '.' -f 1)"
-    export PHP_MINOR="$(php -r 'echo phpversion();' | cut -d '.' -f 2)"
-    export PHP_VERNUM="$(php-config --vernum)"
-    export PHP_PEAR_PHP_BIN=$(phpenv which php)
+  - tests/_ci/export-variables.sh
 
 install:
-  - .ci/install-mysql-5.7.sh
   - tests/_ci/setup-dbs.sh
   - tests/_ci/pear-setup.sh
-  - .ci/install-prereqs.sh
   - tests/_ci/install-re2c.sh
+  - tests/_ci/install-zephir.sh
+  - tests/_ci/install-php-extensions.sh
```

Comment:
I wold like avoid using `tests/_ci` directory. Could you move scripts to `.ci` instead